packages/bot-web-ui/src/stores/blockly-store.ts

```diff
@@ -2,41 +2,68 @@ import { action, observable } from 'mobx';
 import { onWorkspaceResize } from '@deriv/bot-skeleton';
 import { tabs_title } from 'Constants/bot-contents';
 import { storeSetting, getSetting } from 'Utils/settings';
+import RootStore from './root-store';
 
-export default class MainContentStore {
-    constructor(root_store) {
+export interface IBlocklyStore {
+    is_loading: boolean;
+    active_tab: string;
```

Comment:
In `dashboard.store` the `active_tab` is a number, Are you sure it is `string` here?

Reply:
Yes, In previous version the active_tab was string WORKSPACE/CHART

Reply:
this was previous implementation where the value of the active tab was either `WORKSPACE` or `CHART`